packages/konsole-app/src/main.tsx

```diff
@@ -60,14 +59,16 @@ const Main = () => {
  * React app bootstrap (providers and top-level routes).
  * @param config
  */
-const start = (config: IConfig) => {
+const start = async (config: IConfig) => {
   const log = debug('dxos:console:main');
   log('Starting...', {config});
   debug.enable(config.system.debug);
 
+  const dxnsApi = await DxnsApi.create(config.registry.endpoint);
```

Comment:
pass config.registry (likely other config information will be required later).

Reply:
Agree, although other information is already required, I wonder how this was solved.